internal/polling/pollster.go

```diff
@@ -0,0 +1,85 @@
+package polling
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"gophermart/internal/model"
+	"log/slog"
+	"sync"
+	"syscall"
+	"time"
+)
+
+// Нужно опрашивать внешний сервис Acrual с каким-то интервалом до тех пор, пока он не вернет нужный статус по заказу PROCESSED или INVALID
+
+type Pollster struct {
+	incoming    chan int
+	orders      []int
+	stopCh      chan struct{}
+	accrualAddr string
+	store       Store
+}
+
+func NewPollster(accrualAddr string, store Store) *Pollster {
+	incoming := make(chan int)
+	orders := make([]int, 0)
+	stopCh := make(chan struct{})
+	return &Pollster{
+		incoming,
+		orders,
+		stopCh,
+		accrualAddr,
+		store,
+	}
+}
+
+func (p *Pollster) Push(OrderID int) {
+	go func() {
+		p.incoming <- OrderID
+	}()
```

Comment:
Мне кажется тут вызов горутины лишний. По сути он никак не меняет поведение метода.

```suggestion
	p.incoming <- OrderID
```